hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -237,12 +237,12 @@ public final class HoodieMetadataConfig extends HoodieConfig {
           + "metadata table which are never added before. This config determines how to handle "
           + "such spurious deletes");
 
-  public static final ConfigProperty<Boolean> USE_LOG_RECORD_READER_SCAN_V2 = ConfigProperty
-      .key(METADATA_PREFIX + ".log.record.reader.use.scanV2")
+  public static final ConfigProperty<Boolean> ENABLE_OPTIMIZED_LOG_BLOCKS_SCAN = ConfigProperty
+      .key(METADATA_PREFIX + ".optimized.log.blocks.scan.enable")
       .defaultValue(false)
       .sinceVersion("0.13.0")
-      .withDocumentation("ScanV2 logic address all the multiwriter challenges while appending to log files. "
```

Comment:
Do we think it makes sense to put the common sub-string `.optimized.log.blocks.scan.enable` as a constant in `HoodieCompactionConfig` ?